app/controllers/carto/api/visualizations_controller.rb

```diff
@@ -387,10 +391,29 @@ def count
         query += ") AS results2"
         type_counts = Sequel::Model.db.fetch(query, *args).all
 
+        #sample maps counts
+        if samples_user_id
+          user_id = samples_user_id
+          query = "SELECT
+                COUNT(*) AS all,
+                SUM(CASE WHEN likes > 0 THEN 1 ELSE 0 END) AS liked
+              FROM (
+                SELECT results.*, (SELECT COUNT(*) FROM likes WHERE actor=? AND subject=results.id) AS likes FROM (
+                  SELECT v.id, v.type, v.category, v.locked
+                    FROM visualizations AS v
+                    WHERE v.user_id=? AND v.type IN (#{typeList}) AND v.locked=true
+                  ) AS results"
+          args = [user_id, user_id]
```

Comment:
It's a good practice to initialize args right before using them.. so it's a bit more readable. Also, if they're not going to be used elsewhere, maybe you can save the extra variable.

```ruby
args = [user_id, user_id]
sample_type_counts = Sequel::Model.db.fetch(query, *args)
```

or 

```ruby
sample_type_counts = Sequel::Model.db.fetch(query, *[user_id, user_id])
```